Description: Fix pdflatex invocation
 In /usr/share/texlive/texmf-dist/web2c/texmf.cnf, openout_any is set to p
 (paranoid), so that LaTeX does not allow to write out of the current
 directory and its sub-directories.
 .
 Override that in the pdflatex invocation to avoid error:
 Not writing to ../html/examples/group/latex//group__group2.aux
Forwarded: https://github.com/doxygen/doxygen/issues/8226
Author: Paolo Greppi <paolo.greppi@libpf.com>
Last-Update: 2023-08-27

Index: doxygen/doc/CMakeLists.txt
===================================================================
--- doxygen.orig/doc/CMakeLists.txt
+++ doxygen/doc/CMakeLists.txt
@@ -206,11 +206,11 @@ add_custom_target(doxygen_pdf
         COMMAND ${CMAKE_COMMAND} -E copy ${PROJECT_SOURCE_DIR}/doc/doxygen_logo.pdf  .
         COMMAND ${CMAKE_COMMAND} -E chdir ${PROJECT_BINARY_DIR}/latex ${Python_EXECUTABLE} ${PROJECT_SOURCE_DIR}/doc/replace_version.py "${PROJECT_BINARY_DIR}/doc/doxygen_manual.tex" "${PROJECT_BINARY_DIR}/latex/doxygen_manual.tex" "${VERSION}"
         COMMAND ${CMAKE_COMMAND} -E chdir ${PROJECT_BINARY_DIR}/latex ${Python_EXECUTABLE} ${PROJECT_SOURCE_DIR}/doc/replace_version.py "${PROJECT_BINARY_DIR}/doc/manual.sty" "${PROJECT_BINARY_DIR}/latex/manual.sty" "${VERSION}"
-        COMMAND ${PDFLATEX}  -shell-escape doxygen_manual.tex
+        COMMAND ${PDFLATEX}  -cnf-line="openout_any=r" -shell-escape doxygen_manual.tex
         COMMAND ${MAKEINDEX} doxygen_manual.idx
-        COMMAND ${PDFLATEX}  -shell-escape doxygen_manual.tex
+        COMMAND ${PDFLATEX}  -cnf-line="openout_any=r" -shell-escape doxygen_manual.tex
         COMMAND ${MAKEINDEX} doxygen_manual.idx
-        COMMAND ${PDFLATEX}  -shell-escape doxygen_manual.tex
+        COMMAND ${PDFLATEX}  -cnf-line="openout_any=r" -shell-escape doxygen_manual.tex
         DEPENDS ${PROJECT_BINARY_DIR}/doc/manual.sty ${PROJECT_SOURCE_DIR}/doc/doxygen_logo.pdf
         DEPENDS run_doxygen ${PROJECT_SOURCE_DIR}/doc/replace_version.py ${PROJECT_BINARY_DIR}/doc/doxygen_manual.tex
         WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/latex
